tensorflow/tools/ci_build/mac/gpu/pip/run.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+#
+# Usage:
```

Comment:
There are a few reasons, but the most important:
Many people come to me saying they need clarifications on what exactly jenkins runs.
ci_parameterized_build.sh tries to do too much for any bash script, and I usually need to spend many hours with people tracking the script and distilling what it executes with them, so they can finally go back and start debugging problems.

So clarifying the script like this will free up a lot of my time and a lot of people's time when they need to see what is run by the CI, but it will add some code duplication. I think it is a worthy exchange.